core/base/model/AliceSkill.py

```diff
@@ -335,6 +335,11 @@ def dispatchMessage(self, session: DialogSession) -> bool:
 		return self._supportedIntents[session.intentName][1](session=session)
 
 
+	def onMessage(self, session) -> bool:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4677721)